book/accessibility.md

```diff
@@ -2147,14 +2153,69 @@ def announce_text(node):
         text = "Button"
     elif role == "link":
         text = "Link"
+    elif role == "alert":
+        text = "Alert"
     if hasattr(node, "is_focused") and node.is_focused:
         text += " is focused"
     return text
 ```
 
+These alerts are supposed to happen only when the `role` attribute changes
+to alert^[Or the text contexts of the alert changes, but I won't implement
+that.], so we need a way to detect that an attribute. Actually, there isn't
+currently a way to change element attributes other than special ones like
+`style`, so let's first implement that. It will need some runtime code:
+
+``` {.javascript}
+Node.prototype.setAttribute = function(attr, value) {
+    return call_python("setAttribute", this.handle, attr, value);
+}
+```
```

Comment:
I don't think this is something we should act on now, but in a later editing pass, do you think it would make sense to implement `setAttribute` instead of `style` in the compositing chapter?

Reply:
We could yeah.